packages/react-storage/src/components/StorageBrowser/views/hooks/usePaginate.ts

```diff
@@ -25,15 +26,17 @@ export const usePaginate = ({
     setCurrentPage(1);
   }).current;
 
-  return React.useMemo(
-    () => ({
+  return React.useMemo(() => {
+    const isInitialPage = currentPage === 1;
```

Comment:
This is the same change in two PRs, right?

Reply:
yeah, i forgot about that. it's been a while since i opened the other PR. i'll rebase after `useLocationsView` is merged.